apps/electron/package.json

```diff
@@ -15,6 +15,7 @@
     "make-macos-x64": "electron-forge make --platform=darwin --arch=x64",
     "make-windows-x64": "electron-forge make --platform=win32 --arch=x64",
     "make-linux-x64": "electron-forge make --platform=linux --arch=x64",
+    "generate-release-yml": "zx scripts/generate-yml.mjs",
```

Comment:
It seems this script does not exist and `generate-yml.js` does not use zx. You can remove it.

Reply:
removed